lib/ChainwebData/Api.hs

```diff
@@ -49,7 +49,7 @@ deriving instance ToHttpApiData RequestKey
 
 type TxDetailApi = "tx"
     :> QueryParam "requestkey" RequestKey
-    :> Get '[JSON] TxDetail
+    :> Get '[JSON] [TxDetail]
```

Comment:
After reflecting on this some, I think we need to create a new endpoint instead of making this breaking API change to the old endpoint.  Let's call it `txs`.